pkg/azbastion/bastion.go

```diff
@@ -86,7 +87,7 @@ func (t *TunnelSession) Close() error {
 	return nil
 }
 
-func (b *Bastion) NewTunnelSession(targetHost string, port uint16) (*TunnelSession, error) {
+func (b *Bastion) NewTunnelSession(targetHost string, port uint16, scope string) (*TunnelSession, error) {
```

Comment:
are idea to add a default scope?

Reply:
I didn't set it in bastion.go and instead handle it in main.go:
![image](https://github.com/tg123/azbastion/assets/716236/0927a531-15b2-4f5a-a1fb-3c5507582a32)

Reply:
hmm true
agree to move it from lib to user side